tests/unit_tests_formatting.bash

```diff
@@ -51,19 +50,58 @@ function Unit_Test__codebase-formatting()
         report_before='TRUE'
     fi
     if [[ ${#files_with_wrong_formatting[@]} -gt 0 ]]; then
-        if [[ ${report_before} = 'TRUE' ]]; then
+        if [[ "${report_before}" = 'TRUE' ]]; then
             printf '\n'
         fi
         Print_Error \
-            'There are ' --emph "${#files_with_wrong_formatting[@]}" ' file(s) wrongly formatted:'
+            'There are ' --emph "${#files_with_wrong_formatting[@]} Bash" ' file(s) wrongly formatted:'
         for file in "${files_with_wrong_formatting[@]}"; do
             Print_Error -l -- ' - ' \
                 --emph "$(realpath --relative-base="${HYBRIDT_repository_top_level_path}" "${file}")"
         done
         Print_Info '\nRun ' --emph 'Hybrid-handler format' ' to correctly format the codebase.'
     fi
     if ((${#files_with_too_long_lines[@]} + ${#files_with_wrong_formatting[@]} > 0)) \
-        || [[ ${formatter_found} = 'FALSE' ]]; then
+        || [[ "${formatter_found}" = 'FALSE' ]]; then
+        return 1
+    fi
+}
+
+function Unit_Test__codebase-formatting-Python()
+{
+    local formatter_found='FALSE'
+    if hash autopep8 &> /dev/null; then
+        formatter_found='TRUE'
+    else
+        Print_Error 'Command ' --emph 'autopep8' \
+            ' not available, unable to fully check codebase formatting.'
+    fi
+    local list_of_python_files files_with_wrong_formatting file
+    list_of_python_files=(
+        "${HYBRIDT_repository_top_level_path}/python/"*.py
+        "${HYBRIDT_repository_top_level_path}/tests/"**/*.py
```

Comment:
I guess (consistently with the Bash test)

```bash
        "${HYBRIDT_repository_top_level_path}"/**/*.py
```

is enough unless you want to avoid match some files?!